operator/templates/stateful-set.yaml

```diff
@@ -133,6 +135,63 @@ spec:
             - name: jvm-options
               mountPath: /etc/cassandra/jvm.options
               subPath: jvm.options
+        {{ if eq .Params.METRICS_ENABLED "true" }}
+        - name: metrics
```

Comment:
In a similar vein to the previous naming comment, let's s/metrics/prometheus-exporter/` everywhere.